addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/automation.html

```diff
@@ -39,5 +39,19 @@ <H2>Job: report</H2>
 	the strings in the sites list is included. The sites list also supports
 	variables.
 
+	<H2>Job: outputSummary</H2>
```

Comment:
This is not intended to be used by the user, it's an "internal" job to help the migration of the packaged scans to use the AF.